docs/can-stache-element.md

```diff
@@ -206,6 +206,31 @@ document.body.querySelector("button#remove").addEventListener("click", () => {
 @codepen
 @highlight 14-23,only
 
+
+#### Observable class fields
+
+`StachElement` class fields are observables like [can-stache-element/static.props static props],
```

Comment:
Can you link `class fields` to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Class_fields?

Reply:
Done!